javascript/change-notes/2021-03-09-template-object-injection.md

```diff
@@ -0,0 +1,2 @@
+lgtm,codescanning
+* The `js/template-object-injection` query has been added. It highlights servers where an templating engine may allow attacks on the server.
```

Comment:
The wording suggests that we're just flagging the fact that you're using template engine xyz (as opposed to the way you interact with such a template engine).
```suggestion
* The `js/template-object-injection` query has been added. It highlights places where an attacker can pass special parameters to a template engine.
```